xbout/plotting/animate.py

```diff
@@ -100,9 +100,86 @@ def animate_imshow(data, animate_over='t', x='x', y='y', animate=True,
     return imshow_block
 
 
-def animate_line(data, animate_over='t', x='x', y='y', animate=True,
-                 fps=10, save_as=None, sep_pos=None, ax=None, **kwargs):
+def animate_line(data, animate_over='t', animate=True,
+                 vmin='min', vmax='max', fps=10, save_as=None, sep_pos=None, ax=None,
```

Comment:
Unless other strings are acceptable as values for `vmin/vmax`, go with `None` as the default argument